sysutils/salt/Portfile

```diff
@@ -29,31 +29,78 @@ if {$subport eq $name} {
     checksums           rmd160  bc5f5cf85a8a523f30afe815ec1353cf8562ae8b \
                         sha256  6f4ff7cadc97707bbfc42bea0672f359da6b44f1b46ae4192414e9416ab902af
 
-    depends_build       port:py${python.version}-setuptools
-
-    depends_lib-append  port:py${python.version}-crypto \
-                        port:py${python.version}-jinja2 \
-                        port:py${python.version}-msgpack \
-                        port:py${python.version}-pip \
-                        port:py${python.version}-yaml \
-                        port:py${python.version}-tornado \
-                        port:py${python.version}-zmq \
-                        port:swig-python
-
     notes   "Salt startupitems are installed by subports salt-minion, salt-master, salt-syndic, salt-api."
-
-    variant python34 conflicts python35 python36 description {experimental python-3.4 support} {
+    
+    default_variants	+python27 +zmq
+    
+    variant python27 conflicts python34 python35 python36 description {experimental python-3.5 support} {
+		python.default_version 27
+		depends_build	    port:py${python.version}-setuptools
+	
+		depends_lib-append  port:py${python.version}-yaml \
+							port:py${python.version}-jinja2 \
```

Comment:
New text editor.. hadn't noticed. I need to fix its TCL whitespace settings. Vi modes not obeyed :(